code/src/main/resources/site/i18n/phrases_no.properties

```diff
@@ -6,35 +6,50 @@ site.testmode-help = Aktiver testmodus hvis du alltid vil vise forespørselsboks
 site.identifier = Identifikator
 site.identifier-help = Skriv inn tekst og/eller nummer for å identifisere gjeldende forespørsel. Alle skjemaer som sendes inn av besøkende vil bli koblet til denne identifikatoren. Når du vil opprette en ny forespørsel, endre identifikatoren til noe nytt og unikt. Vennligst ikke bruk mellomrom eller spesialtegn annet enn "-".
```

Comment:
Seeing some Ã¸ and Ã¥ all over here. Is it some text encoding choice on your IDE that is not supported here? Could this be an issue, or can I ignore this? :)